docs/index.md

```diff
@@ -1,50 +1,34 @@
 ---
 title: Home
-nav_order: 1
+layout: home
+hero:
+    name: Openrouteservice Backend Documentation
+    tagline: Smart mobility made easy
```

Comment:
We might want a nicer tagline here^^

Reply:
😑 then suggest a better one... that's the best i came up with.